ansible/group_vars/all

```diff
@@ -64,6 +64,17 @@ insight_port: 3001
 # DashCore Docker image
 dashd_image: dashpay/dashd
 
+users_to_manage:
+- user: "ubuntu"
+  group: "ubuntu"
+  dir: "/home/ubuntu"
+- user: "root"
+  group: "root"
+  dir: "/root"
+- user: "dash"
+  group: "dash"
+  dir: "/dash"
```

Comment:
Let's specify the dash user using vars like `{{ dashd_home }}` so we don't have to define them twice (also on line 78-80).

Reply:
Yes, in theory. I do plan to remove the dash user in the future if there is no solid justification for keeping it,  but this can keep it simpler for now.